include/sys/syscall_lookup.h

```diff
@@ -41,10 +41,12 @@ SYSCALL_LOOKUP(sched_getparam,             2)
 SYSCALL_LOOKUP(sched_getscheduler,         1)
 SYSCALL_LOOKUP(sched_lock,                 0)
 SYSCALL_LOOKUP(sched_lockcount,            0)
+SYSCALL_LOOKUP(sched_lock_wo_note,         0)
```

Comment:
remove no trace version from syscall

Reply:
sched_lock_wo_note may be called in userspace 
![image](https://github.com/user-attachments/assets/19e21ab2-2dc3-47e5-88d9-90d8c745b605)